### ISSUE TITLE
Cluster setup: use database time instead of instance system time

### ISSUE THREAD

**Issue by vtylyk** [quartznet/quartznet#435]
Hello,

Is there a specific reason why in clustering setup the value of the "**LAST_CHECKIN_TIME**" column for each cluster node is filled in with system time (**DateTimeOffset.UtcNow**) instead of using database time (**GetUtcDate()**)? 

This would help to avoid clock skew issues which can happen with clustering setup, since database time would be a single source of truth. Or am I missing something?

**Comment by lahma** [comment]
GetUtcDate is SQL Server specific, but there's a bigger concern behind here. Every cluster node is expected to be time-synchronized due to the fact that all polling operations against database use times based on callee, the scheduler.

So the last check-in time is just one of the columns that depends on the fact that all servers need to have time synchronized. Triggers are checked against scheduler server time when retrieved from db to be executed.

So in short, time logic is in the scheduler, not the database. Due to this, please just make sure you have NTP daemon running on all hosts.
